cmd/kubeadm/app/phases/addons/addons_test.go

```diff
@@ -86,3 +92,71 @@ func TestCompileManifests(t *testing.T) {
 		}
 	}
 }
+
+// This unit test exists because tolerations weren't decoded correctly by kuberuntime.DecodeInto
```

Comment:
@caesarxuchao It was moved from an annotation to a field, and the field was undetectable/unparsable by the API Decoder...